src/test/unit/services/AuthServiceTests.ts

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { expect } from 'chai';
+import { describe, it } from "node:test";
+import { getTokenByloggingIn, URL } from '../../../main/services/AuthService';
+import { LoginRequest } from "../../../main/models/LoginRequest";
+
+const loginResquest: LoginRequest = {
+  email: "adam@random.com",
+  password: "pass123"
+}
+
+const mock = new MockAdapter(axios);
+
+describe('AuthService', function () {
+
+    describe('getToken', function () {
+      it('should get Jwt token', async () => {
+        mock.onPost(URL).reply(200);
+
+        const result = await getTokenByloggingIn(loginResquest);
+
+        expect(result).toString.length > 100;
```

Comment:
what is this checking? you should be returning a token from the mock and expecting that value to be returned from the service call